routes/web.php

```diff
@@ -49,6 +57,13 @@
         Route::get('/profile/edit', [ProfileController::class, 'edit'])->name('profile.edit');
         Route::patch('/profile/update', [ProfileController::class, 'update'])->name('profile.update');
     });
+  
+//user event
+Route::group(['prefix' => 'events' , 'as' => 'events.'], function(){
+    Route::get('/event', [EventController::class, 'show'])->name('show');
```

Comment:
no need to specify the word `event` here since we already have a prefix `events` in our route group. It should be just `/`